examples/PublishHelloWorldCAN/README.md

```diff
@@ -0,0 +1,33 @@
+# PublishHelloWorldCAN example
+
+This example will show how to send data to the DDS World creating a client publisher using a CAN bus.
+In order to compile this example, the following profiles should be enabled:
+
+- `UCLIENT_PROFILE_CAN`
+
+To run the example, a virtual CAN FD bus can be created with the following commands:
+```
+ip link add dev vcan0 type vcan
+ip link set vcan0 mtu 72
+ip link set dev vcan0 up
+```
+
+## Usage
+1. Run an agent in can mode, for example, *2021*: `MicroXRCEAgent canfd --dev vcan0 --id 0x00000001`.
```

Comment:
What is the `for example 2021`?

Reply:
All the examples add a reference year for the command ([example](https://github.com/eProsima/Micro-XRCE-DDS-Client/blob/0f4019b01cc1851c92d7a9e4e19ec2ed6c7d4729/examples/MultiSessionHelloWorld/README.md)).

Maybe we can update the commands if needed and remove this year references.

Reply:
```suggestion
1. Run an agent in can mode, for example: `MicroXRCEAgent canfd --dev vcan0 --id 0x00000001`.
```